src/web/central_command/client/components/AXUI.jsx

```diff
@@ -1667,18 +1703,22 @@ export default class AXUI extends React.Component {
 				color = unselectedColor
 			}
 
+
 			let goals = missions[botId]?.plan?.goal || []
 
 			let transformed_pts = goals.map((goal) => {
 				return equirectangular_to_mercator([goal.location.lon, goal.location.lat])
 			})
 
+			// console.log('transformed_pts');
```

Comment:
Are these for debugging? If so, maybe we just add a comment above to explain what they provide.